README.md

```diff
@@ -33,19 +35,393 @@ yarn chain
 > in a second terminal window, 🛰 deploy your contract (locally):
 
 ```sh
-cd {challengeName}
+cd challenge-4-dex
 yarn deploy
 ```
 
 > in a third terminal window, start your 📱 frontend:
 
 ```sh
-cd {challengeName}
+cd challenge-4-dex
 yarn start
 ```
 
 📱 Open http://localhost:3000 to see the app.
 
-## Checkpoint {num}: Create all the required Checkpoints for the challenge
+## ⛳️ Checkpoint 1: 🔭 The Structure 📺
+Navigate to the Debug Contracts tab and you should see two smart contracts displayed called DEX and Balloons.
+
+👩‍💻 Rerun yarn deploy whenever you want to deploy new contracts to the frontend (run yarn deploy --reset for a completely fresh deploy if you have made no contract changes).
+
+Balloons.sol is just an example ERC20 contract that mints 1000 $BAL to whatever address deploys it. DEX.sol is what we will build in this challenge and you can see it starts with a SafeMath library to help us prevent overflows and underflows and also tracks a token (ERC20 interface) that we set in the constructor (on deploy).
+
+Below is what your front-end will look like with no implementation code within your smart contracts yet. The buttons will likely break because there are no functions tied to them yet!
+
+⭐️ Also note that there is no curve until you uncomment the specific lines of code at the end of `hardhat/deploy/00_deploy_your_contract.ts`.
+
+![2023-07-13 (2)](https://github.com/mertcanciy/se-2-challenges/assets/59885513/1b6e8f59-c549-40b8-982e-e0bec0239117)
+
+
```

Comment:
there's no this block of text, why?

<img width="1098" alt="image" src="https://github.com/scaffold-eth/se-2-challenges/assets/25638585/9ebb3350-d6d4-4fb1-9d27-5a8a501b48d9">